Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM docker.io/nginx
+FROM ubuntu:latest
```

Comment:
Can we use some light weight container or RH UBI image?

Reply:
Added alpine based image, `entrypoint.sh` command need to be changed a/cly